starlite/openapi/controller.py

```diff
@@ -44,6 +44,49 @@ class OpenAPIController(Controller):
     """
     URL to download a favicon from.
     """
+    redoc_google_fonts: bool = True
+    """
+    Google CDN Fonts in Redoc, use False for use in isolated environment
+    """
+    redoc_js_url: str = f"https://cdn.jsdelivr.net/npm/redoc@{redoc_version}/bundles/redoc.standalone.js"
+    """
+    Specify redoc_js_url for use in isolated environment
+    Default reverts to as used in redoc_version
```

Comment:
```suggestion
```

Reply:
Changed to your suggestion. Thanks :)
also changed it for the others.